lib/character.rb

```diff
@@ -0,0 +1,9 @@
+class Character
+  attr_reader :name, :actor, :salary
+
+  def initialize(name:, actor:, salary:)
```

Comment:
Good job using keyword arguments here; they're less common in Ruby than other languages, but great that you found a use for them here.